starter_code/app.js

```diff
@@ -10,12 +10,33 @@ app.set('view engine', 'hbs');
 app.set('views', __dirname + '/views');
 app.use(express.static(path.join(__dirname, 'public')));
 
-
+hbs.registerPartials(__dirname + '/views/partials')
 
 app.get('/', (req, res, next) => {
   res.render('index');
 });
 
 
+app.get('/beers', (req, res, next) => {
+  punkAPI.getBeers()
+  .then(beers => {
+    res.render('beers', {x:beers});
+    console.log(beers)
+  })
+  .catch(error => {
+    console.log(error)
+  })
+});
+
+app.get('/random-beers', (req, res, next) => {
+  punkAPI.getRandom()
+  .then(beers => {
+    res.render('random-beers',  {y:beers});
+  })
```

Comment:
Instead of `x` and `y` perhaps name these variables `allBeers` and `randomBeers` so its clear why different partials are used for different data.